packages/plugin/fuses/test/FusesPlugin_spec_slow.ts

```diff
@@ -49,13 +49,14 @@ describe('FusesPlugin', () => {
       platform: process.platform,
     });
 
+    const args: string[] = process.platform === 'linux' ? ['-v', '--no-sandbox'] : ['-v'];
```

Comment:
Is this just for Snapcraft, or do we also need this for Flatpak?

Reply:
This should be for headless linux tests in general i believe? ([source](https://bugs.chromium.org/p/chromium/issues/detail?id=638180))